lite/examples/object_detection/ios/ObjectDetection/ModelDataHandler/ModelDataHandler.swift

```diff
@@ -102,13 +92,11 @@ class ModelDataHandler: NSObject {
 
     // Specify the options for the `Interpreter`.
     self.threadCount = threadCount
-    var options = Interpreter.Options()
-    options.threadCount = threadCount
+    let options = ObjectDetectorOptions(modelPath: modelPath)
```

Comment:
Please add options in the bottom sheet UI to:
- Allow user to set the scoreThreshold. Select value from 0.0 to 0.9, with 0.1 steps.
- Allow user to set the maxResults. Select value from 1 to 10, with 1 steps.

Refer to this Android UI as an example.
![image](https://user-images.githubusercontent.com/5379361/170201860-dcbe523e-c471-4d87-9894-41c923c3ac6e.png)